spec/lib/api/api_config_spec.rb

```diff
@@ -29,7 +39,7 @@
             cfg[action_type].each do |_, method_cfg|
               method_cfg.each do |action_cfg|
                 next unless action_cfg[:identifier]
-                Array(action_cfg[:identifier]).each { |id| set.add(id) }
+                Array(action_cfg[:identifier]).each { |id| add_identifier_to_set(action_cfg[:identifier], set) }
```

Comment:
probably don't need this as it already deals with the Array

Reply:
changed 👍